definitions/divorce/json/CaseEventToFields/CaseEventToFields-deemed-and-dispensed-nonprod.json

```diff
@@ -51,5 +51,69 @@
     "PageColumnNumber": 1,
     "PageShowCondition": "ServiceApplicationGranted=\"No\"",
     "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "04/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "ServicePayment",
+    "CaseFieldID": "ServiceApplicationPayment",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "ServiceApplicationPayment1",
```

Comment:
Can we have a more specific name? Isn't this specific for deemed and dispensed?
Or is it really a generic event for any service payment?

Reply:
I believe it is generic

Reply:
Cool. Can you please confirm with Jeremy?

Reply:
As long as it's unique it's a valid name.

I think we usually use pattern: $nameOfEvent + $oderOfPageInThisEventFlow,

eg:
`"CaseEventID": "serviceApplicationReceived",` -> `"PageID": "serviceApplicationReceived1",`

Reply:
☝️